webroot/src/components/Forms/InputNumber/InputNumber.ts

```diff
@@ -0,0 +1,29 @@
+//
+//  InputNumber.ts
```

Comment:
For this, I was initially wanting to make the enforcement of only numerals in the input to be part of this component. Ultimately, this broke from the pattern we have used and because I was behind on the ticket I didn't want to think through the best way to do this on a pattern level + properly interact with joi validations.

I left the component in because I may want to do this in the future. Im fine just removing the numberinput component for now if thats your preference.

Reply:
Let's remove it and just use InputText for now. In your `validation` you can just use [`Joi.number()....`](https://joi.dev/api/?v=17.13.3#number)